src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello from\n" + logo);
```

Comment:
Perhaps, you could use "**\t** Hello from...." to print out indented text to the user.